WcaOnRails/db/seeds/formats.seeds.rb

```diff
@@ -1,9 +1,11 @@
 # frozen_string_literal: true
-sql = "INSERT INTO `Formats` (`id`, `name`) VALUES
-('1', 'Best of 1'),
-('2', 'Best of 2'),
-('3', 'Best of 3'),
-('a', 'Average of 5'),
-('m', 'Mean of 3');"
-connection = ActiveRecord::Base.connection()
-connection.execute(sql)
+sql = "INSERT INTO Formats
+(id, name, sort_by, sort_by_second, expected_solve_count,
+trim_fastest_n, trim_slowest_n)
+VALUES
+('1', 'Best of 1', 'single', 'average', 1, 0, 0),
+('2', 'Best of 2', 'single', 'average', 2, 0, 0),
+('3', 'Best of 3', 'single', 'average', 3, 0, 0),
+('a', 'Average of 5', 'average', 'single', 5, 1, 1),
+('m', 'Mean of 3', 'average', 'single', 3, 0, 0);"
+ActiveRecord::Base.connection.execute(sql)
```

Comment:
I think that all seeds should be written in ruby (Rails DSL), to call callbacks and ensure that we don't create invalid objects. We used SQL before, because there were no models =)